xray/plot/facetgrid.py

```diff
@@ -127,14 +129,17 @@ def __init__(self, data, col=None, row=None, col_wrap=None,
                 ncol = col_wrap
             nrow = int(np.ceil(nfacet / ncol))
 
+        # Set the subplot kwargs
+        subplot_kws = {} if subplot_kws is None else subplot_kws.copy()
```

Comment:
I don't think you need to copy `subplot_kws`? `plt.subplots` is a pure function (at least it should be!).